platforms/Windows/ide/ide.wxs

```diff
@@ -37,6 +37,14 @@
         <File Source="$(TOOLCHAIN_ROOT)\usr\lib\sourcekitdInProc.lib" />
       </Component>
 
+      <Component Directory="_usr_lib">
+        <File Source="$(TOOLCHAIN_ROOT)\usr\lib\SwiftSourceKitClientPlugin.dll" />
+      </Component>
+
+      <Component Directory="_usr_lib">
+        <File Source="$(TOOLCHAIN_ROOT)\usr\lib\SwiftSourceKitClientPlugin.dll" />
+      </Component>
```

Comment:
Why is this duplicated?

Reply:
You mean, why I’ve got two `Component` clauses pointing at `_usr_lib`? Looking at the other install locations, I couldn’t find any examples of two `File` entries in one component, so I thought that that’s the pattern.

Also note that this is SwiftSourceKit**Client**Plugin.dll and not SwiftSourceKitPlugin.dll

Reply:
Sorry, maybe I'm misreading it, but the source file that is being installed is the _same_ file, so you have two entries for the same file.

Reply:
Oh, whoops, they are indeed the same. Thanks for noticing the copy-paste error.